material_3_demo/lib/color_palettes_screen.dart

```diff
@@ -303,13 +303,13 @@ class ColorSchemeView extends StatelessWidget {
           children: [
             ColorChip(
               label: 'background',
```

Comment:
We can actually remove these chips whose colors have been deprecated:)

Reply:
Feel free to adopt this change and adapt it to suit your needs. I was just getting in early to see what the required changes for the next stable release might look like.

Reply:
Just pushed a small change to remove the deprecated color chips! :)